source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst

```diff
@@ -49,13 +70,13 @@ Upgrade Elasticsearch
 
     .. code-block:: console
 
-      # yum install elasticsearch-6.8.6
+      # yum install elasticsearch-6.x.6
```

Comment:
This `x` doesn't work. This branch must be updated with the 3.11 latest changes `6.8.7`.

Reply:
Done!